apps/portal/src/common/constants/notification.ts

```diff
@@ -75,4 +76,11 @@ export const ChannelTypeMap = {
     providerIcon: 'assets/icons/kaps.png',
     style: 'sms',
   },
+  [ChannelType.PUSH_SNS]: {
+    serviceIcon: 'assets/icons/pushnotification.svg',
+    altText: 'Push Notification',
+    providerName: 'AWS Push SNS',
```

Comment:
We can probably set providerName as AWS SNS only. Since in the dropdown, we append the altText:
![image](https://github.com/OsmosysSoftware/osmo-x/assets/108812833/8d1c9df8-8e4e-4d03-ab66-9d3819613efe)

Also, to remain consistent with how we only use provider name for other channels as well.

Icons and colours are fine.

Reply:
fixed